### ISSUE TITLE
Parallels 16 download no longer including .app

### ISSUE THREAD

**Issue by skoobasteeve** [autopkg/homebysix-recipes#405]
Seems like with the most recent update to Parallels 16 they aren't including the hidden full app bundle in their download, only the net installer:

<img width="561" alt="Screen Shot 2021-04-16 at 10 00 29 AM" src="https://user-images.githubusercontent.com/36998292/115035688-d3c53100-9e9a-11eb-99c5-c242500725b3.png">

This breaks the download recipe in its current form. I reached out to Parallels to see if they can fix it or provide an alternate link, I'll update if I hear back.

**Comment by skoobasteeve** [comment]
There are some direct download links here that work with the recipe:

https://www.parallels.com/products/business/download/

The problem is that they have separate downloads for M1 and Intel:

https://www.parallels.com/directdownload/pd16/m1/
https://www.parallels.com/directdownload/pd16/intel/

Does it make sense to add a new key to the recipe called PROCESSOR or similar and then apply that to the download URLs? Example below:

``` xml
	<key>Identifier</key>
	<string>com.github.homebysix.download.ParallelsDesktop</string>
	<key>Input</key>
	<dict>
		<key>MAJOR_VERSION</key>
		<string>15</string>
		<key>NAME</key>
		<string>Parallels Desktop</string>
                <key>PROCESSOR</key>
                <string>m1</string>
	</dict>
       .......
       .......
			<dict>
				<key>filename</key>
				<string>%NAME%-%MAJOR_VERSION%-%PROCESSOR%.dmg</string>
				<key>url</key>
				<string>https://www.parallels.com/directdownload/pd16/%PROCESSOR%/</string>
			</dict>
			<key>Processor</key>
			<string>URLDownloader</string>
```

I imagine this is going to be a problem for a lot of apps going forward and developers create separate packages for arm and Intel. What do you think?

**Comment by mpanighetti** [comment]
Suggested the changes in #407. I didn't end up changing the filename in order to maintain support for versions 15 and older.

**Comment by homebysix** [comment]
Fixed in #407.
